src/mscorlib/src/System/AppDomainSetup.cs

```diff
@@ -196,11 +185,7 @@ public String TargetFrameworkName
         {
             get
             {
-                return _TargetFrameworkName;
-            }
-            set
-            {
-                _TargetFrameworkName = value;
+                return null;
```

Comment:
Some of those fields are set from the runtime -- I am not sure this should be removed.